test-data/unit/fine-grained.test

```diff
@@ -1230,6 +1230,31 @@ def f() -> None: pass
 ==
 main:2: error: Module 'a' has no attribute 'f'
 
+[case testRefreshTyping]
+from typing import Sized
+from c import A
+
+# Force typing to get refreshed by using a protocol from it
+x : Sized = A()
+
+[file c.py]
+class D:
+    def __len__(self) -> int: return 0
+A = D
+[file c.py.2]
+class C:
+    def __len__(self) -> int: return 0
+A = C
+[file z.py]
+from typing import Tuple, List
+T = List[int]
+[file z.py.2]
+from typing import Tuple, List
+T = List[int]
```

Comment:
Few questions here:
* Why `Tuple` also needs to be imported
* It looks like there are no changes in `z.py` between runs, is this intentional?
* Should `z` be imported somewhere.

Reply:
`Tuple` doesn't need to be.
Should be a change, got busted while iterating.
Doesn't need to be imported in fine-grained tests.